clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -365,4 +365,142 @@ public void createServerGroup_errorIfCreateServiceFails()
         String.format("Failed to observe task failure after %s seconds", TASK_RETRY_SECONDS),
         TASK_RETRY_SECONDS);
   }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given description w/ inputs, EC2 launch type, and legacy target group fields "
+          + "with no load balancing successfully submit createServerGroup operation"
+          + "\n===")
+  @Test
+  public void createServerGroup_InputsEc2LegacyTargetGroupTestWithoutLoadBalacingTest()
```

Comment:
please fix this test name and the display text (how can it have "legacy target group fields" AND "no load balancing"?)

Reply:
Resolved.